### PR TITLE
Fixes #18446 - Hide label fields when the value is empty or nil

### PR DESCRIPTION
**Pull request by tstrachota**

### CODE REVIEW

File: lib/hammer_cli/output/fields.rb [hunk 1]
```diff
@@ -85,7 +85,7 @@ class Label < ContainerField
     def display?(value)
       return true if not hide_blank?
 
-      fields.any? do |f|
+      !(value.nil? || value.empty?) && fields.any? do |f|
```
Comment (ares) [inline lib/hammer_cli/output/fields.rb]: Isn't this equivalent to `value.present?`

Comment (ares) [inline lib/hammer_cli/output/fields.rb]: oh, hammer probably does not depend on active support gem, sorry, dismiss :-)

**Comment by shlomizadok** [comment]
LGTM :+1:

**Comment by tstrachota** [comment]
Thanks guys, merging

**Pull request merged**